api/models/Responsibility.js

```diff
@@ -0,0 +1,72 @@
+const RESP_ADMINISTRATION = 'Administration'
```

Comment:
i noticed you are using Strings for all these. we need to make sure people dont add new responsibilities with the same name as these then